frontends/p4/fromv1.0/programStructure.cpp

```diff
@@ -59,7 +59,14 @@ const IR::Annotations*
 ProgramStructure::addNameAnnotation(cstring name, const IR::Annotations* annos) {
     if (annos == nullptr)
         annos = IR::Annotations::empty;
-    return annos->addAnnotationIfNew(IR::Annotation::nameAnnotation, new IR::StringLiteral(name));
+    return annos->addAnnotationIfNew(IR::Annotation::nameAnnotation,
+                                     new IR::StringLiteral(name));
+}
+
+const IR::Annotations*
+ProgramStructure::addGlobalNameAnnotation(cstring name,
```

Comment:
Yeah, at some point we need to synchronize the way this stuff works with the spec. It's currently somewhat out of date.